_redirects

```diff
@@ -137,7 +138,8 @@ docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab /docs/deploy/dbt-cloud-job
 /docs/dbt-cloud/cloud-quickstart /docs/get-started/getting-started/set-up-dbt-cloud 301
 /docs/cloud-quickstart /docs/dbt-cloud/cloud-quickstart 301
 /docs/dbt-cloud/cloud-configuring-dbt-cloud /docs/get-started/getting-started/set-up-dbt-cloud 301
-/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database /docs/get-started/connect-your-database 301
+/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database /docs/cloud/connect-data-platform/about-connections 301
+/docs/get-started/connect-your-database /docs/cloud/connect-data-platform/about-connections 301
```

Comment:
i don't think this line is needed -- you added it above on line 19

```suggestion

```

Reply:
should [line 76 and 77](https://github.com/dbt-labs/docs.getdbt.com/blob/a49e28bad7be610d107710844691c26f381093b3/_redirects#L76-L77) also have redirects?

Reply:
also, [lines 141 - 142](https://github.com/dbt-labs/docs.getdbt.com/blob/a49e28bad7be610d107710844691c26f381093b3/_redirects#L76-L77) look to be duplicates of your new lines in 19 and 20. sugg removing one of these sets

Reply:
Added redirects for 76/77 and moved line 20 down